runtime/src/invoke_dpe.rs

```diff
@@ -72,6 +74,7 @@ impl InvokeDpeCmd {
                     &nb,
                     &nf,
                     None,
+                    Some(ueid),
                 ),
             };
```

Comment:
In the `DeriveContext` matcher below, I wonder if we should fail the command if PAUSER is not PL0 and `export_cdi` is set. Since the exported CDI isn't usable for PL1 anyway.

Reply:
Done!